fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -913,6 +914,11 @@ private static List<RewriteJob> getWholeTreeRewriteJobs(
                             )
                     ));
                 }
+                rewriteJobs.add(
```

Comment:
use needSubPathPushDown to control do it or not, let nested type same with variant?